clis/daemon-common/src/daemon.rs

```diff
@@ -2,10 +2,10 @@
 
 use std::path::PathBuf;
 
-use anyhow::{self, Result};
+use anyhow::Result;
 use sysinfo::{ProcessRefreshKind, RefreshKind, System};
 
-use crate::TCLID_WD_PATH;
+use daemonize::Daemonize;
 
 /// Enum for figuring out the current process after the daemon is forked away
 /// from the API by the daemonization.
```

Comment:
I wonder, does it really make sense to use the same process (binary) for daemon and for client? :thinking: